### ISSUE TITLE
code problem

### ISSUE THREAD

**Issue by jjkinga** [sc22mc/DocFusion#4]
I'm truly impressed by your paper! I was particularly intrigued by the Gaussian-Kernel Cross-Entropy Loss and believe it's a key innovation.

I was wondering if there's any possibility that the training code could be made open source? If not, perhaps you might consider sharing just the code for the Gaussian-Kernel Cross-Entropy Loss? Either would be greatly appreciated as I'm keen to learn more about its practical application.
@sc22mc

**Comment by SII-sc22mc** [comment]
The relevant code should be Line 2102 in https://github.com/sc22mc/DocFusion/blob/main/DocFusion_base/modeling_docfusion.py
